packages/exoflex/src/components/RichRadio/RichRadioItem.tsx

```diff
@@ -0,0 +1,82 @@
+import React from 'react';
+import {
+  StyleProp,
+  ViewStyle,
+  TextStyle,
+  TouchableOpacity,
+  StyleSheet,
+} from 'react-native';
+
+import Text from '../Text';
+
+import useTheme from '../../helpers/useTheme';
+
+export type RichRadioItemProps = {
+  label: string;
+  selectedColor?: string;
+  selected?: boolean;
+  uppercase?: boolean;
+  firstItem?: boolean;
+  onPress: () => void;
+  style?: StyleProp<ViewStyle>;
+  textStyle?: StyleProp<TextStyle>;
+  firstItemCustomStyle?: StyleProp<ViewStyle>;
+  testID?: string;
+};
+
+export default function RichRadioItem(props: RichRadioItemProps) {
+  let {
+    label,
+    selected,
+    uppercase,
+    firstItem,
+    selectedColor,
+    onPress,
+    style,
+    textStyle,
+    firstItemCustomStyle,
+    testID,
+  } = props;
+  let { colors, style: themeStyle } = useTheme();
+
+  let selectedStyle = {
+    borderColor: selectedColor || colors.primary,
+  } as ViewStyle;
+  let combinedItemStyle = StyleSheet.flatten([
+    styles.item,
+    firstItem && (firstItemCustomStyle || styles.firstItem),
+    { borderColor: colors.border },
+    selected && selectedStyle,
+    themeStyle?.richRadioItem?.style,
+    style,
+  ]) as ViewStyle;
```

Comment:
I think the `firstItemCustomStyle` should have higher priority, overriding the `style` props.